Easy/CountAndSay_38.java

```diff
@@ -0,0 +1,49 @@
+
+/**
+ * The count-and-say sequence is the sequence of integers beginning as follows:
+   1, 11, 21, 1211, 111221, ...
+
+ * 1 is read off as "one 1" or 11.
+ * 11 is read off as "two 1s" or 21.
+ * 21 is read off as "one 2, then one 1" or 1211.
+ * Given an integer n, generate the nth sequence.
+
+ * Note: The sequence of integers will be represented as a string.
+ * Note: brute way: loop string
+ * @author chenshuna
+ *
+ */
+
+public class countAndSay {
+    public static String countAndSay(int n) {
+        String restult = "1";
+        for(int i=1;i<n;i++){
+            restult = countAndSayArray(restult);
+        }
+        return restult;
+    }
+     public static String countAndSayArray(String cas) {
+         int n = cas.length();
+         int count = 1;
+         int point = 0;
+         StringBuilder res = new StringBuilder();
+         while(point < n){
+             while((point+count) < n && cas.charAt(point) == cas.charAt(point+count)){
+                 count++;
+             }
+             res.append(count + "");
+             res.append(cas.charAt(point) + "");
+             point = point + count;
+             count = 1;
+         }
+         
+         return res.toString();
+         
+     }
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```

Comment:
我用命令行时总是deny，然后我就用Github 里 new file 直接提交。但是它会自动变成 tab 8 还有 public static void main 对不齐的情况。。。自动的。。。怎么办。。。

On Mar 14, 2016, at 5:14 PM, Kiyan <notifications@github.com> wrote:

> In Easy/CountAndSay_38.java:
> 
> > +         while(point < n){
> > +             while((point+count) < n && cas.charAt(point) == cas.charAt(point+count)){
> > +                 count++;
> > +             }
> > +             res.append(count + "");
> > +             res.append(cas.charAt(point) + "");
> > +             point = point + count;
> > +             count = 1;
> > +         }
> > +         
> > +         return res.toString();
> > +         
> > +     }
> > +
> > +	public static void main(String[] args) {
> > +		// TODO Auto-generated method stub
> please fix the indentation here and remove this line
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/FreeTymeKiyan/LeetCode-Sol-Res/pull/52/files#r56098072
>